_mobile/ios.md

```diff
@@ -194,6 +193,37 @@ In the build settings, search for **other linker flags**.  Add a custom linker f
 
 Currently, the iOS framework uses the Pytorch C++ front-end APIs directly. The C++ document can be found here https://pytorch.org/cppdocs/. To learn more about it, we recommend exploring the [C++ front-end tutorials](https://pytorch.org/tutorials/advanced/cpp_frontend.html) on PyTorch webpage. In the meantime, we're working on providing the Swift/Objective-C API wrappers to PyTorch.
 
+
+### Custom build
+
+Starting from 1.4.0, PyTorch supports custom build. You can now build the PyTorch library that only contains the operators needed by your model. To do that, follow the steps below
+
+1. Verify your PyTorch version is 1.4.0 or above. You can do that by checking the value of `torch.__version__`.
+2. To dump the operators in your model, we need to run a few lines of Python code
```

Comment:
Rephrase so stays in second person (you) - " To dump the operators in your model, run the following lines of Python code: